package-lock.json

```diff
@@ -195,9 +132,9 @@
       "dev": true
     },
     "@types/node": {
-      "version": "12.7.12",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-12.7.12.tgz",
-      "integrity": "sha512-KPYGmfD0/b1eXurQ59fXD1GBzhSQfz6/lKBxkaHX9dKTzjXbK68Zt7yGUxUsCS1jeTy/8aL+d9JEr+S54mpkWQ==",
+      "version": "13.1.2",
+      "resolved": "https://registry.npmjs.org/@types/node/-/node-13.1.2.tgz",
```

Comment:
Just to double check - does this means that you're using Node v13.1.2 locally? Please lets make this a conscientious choice by upgrading `.nvmrc`, or simply remember to do `nvm use` before installing dependencies (or at least `nvm use && npm i` before committing).

Reply:
Or better yet, set up your shell to [automatically switch to the correct version](https://github.com/nvm-sh/nvm#deeper-shell-integration) :)

Reply:
(oh, I'm definitely doing that)

Reply:
Yeah I meant @michielbdejong ;)

Reply:
OK, odd. I added that now. Maybe it was because I ran `npm upgrade`